sonar-helm-for-iac/converters/io-handling_test.go

```diff
@@ -80,11 +82,59 @@ func Test_read_one_file_with_trailing_newline(t *testing.T) {
 func Test_read_two_files(t *testing.T) {
 	scanner := bufio.NewScanner(strings.NewReader("file1\n2\nline1\nline2\nfile2\n1\nline3\nEND"))
 	stdinReader := StdinReader{}
-	contents := stdinReader.ReadInput(scanner)
+	contents, _ := stdinReader.ReadInput(scanner)
 
 	assert.Equal(t, 2, len(contents))
 	assert.Equal(t, "file1", contents[0].Name)
 	assert.Equal(t, "line1\nline2", contents[0].Content)
 	assert.Equal(t, "file2", contents[1].Name)
 	assert.Equal(t, "line3", contents[1].Content)
 }
+
+func Test_should_stop_if_zero_length(t *testing.T) {
+	scanner := bufio.NewScanner(strings.NewReader("file1\n0\n"))
```

Comment:
Clarification question: Is it always a given that we don't end in `\n` in a template content? Otherwise we would fail quite often (what we don't do, so I think that's a given?)

Reply:
Ok, looking at HelmEvaluator, we're ending with `END%n`, that's why